inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/CommonOperateService.java

```diff
@@ -74,8 +74,12 @@ public String getSpecifiedParam(String key) {
             case Constant.PULSAR_SERVICEURL: {
                 clusterEntity = getThirdPartyCluster(Constant.MIDDLEWARE_PULSAR);
                 if (clusterEntity != null) {
-                    params = gson.fromJson(clusterEntity.getExtParams(), Map.class);
-                    result = params.get(key);
+                    if (key.equals(Constant.PULSAR_SERVICEURL)) {
```

Comment:
In the `case` statement, you have already checked the key matches `Constant.PULSAR_SERVICEURL`, so it's recommended to remove this `if`, and do your work in `case Constant.PULSAR_ADMINURL` and `case Constant.PULSAR_SERVICEURL`.

Reply:
okkk